src/test/java/com/werfen/report/CompareExcelFiles.java

```diff
@@ -0,0 +1,105 @@
+package com.werfen.report;
+
+
+import org.apache.poi.ss.usermodel.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class CompareExcelFiles {
+
+    public void verifyIfExcelFilesHaveSameNumberAndNameOfSheets(Workbook workbook1, Workbook workbook2) {
+
+        int sheetsInWorkbook1 = workbook1.getNumberOfSheets();
```

Comment:
These methods are great, but they force the consumers of this utility to do multiple calls to check if 2 workbooks are equal. I would suggest going for a single method "assertWorkbookEquals(actual, toTest)" that calls a submethod like assertSheetEquals(actual, toTest). This can be further applied for other details to be checked.